lms/djangoapps/grades/signals/handlers.py

```diff
@@ -233,7 +233,7 @@ def listen_for_grade_calculation_to_send_push(sender, user, course_grade, course
             "courseId": course_key.to_deprecated_string(),
             "lastlogin": str(user.last_login or ''),
             "percentageOfcompletion": percentageOfcompletion,
-            "duration": str(duration),
+            "duration": int(duration // 3600),
```

Comment:
It will round to flow every time.
Change it to `int(round(duration / 3600))` or something like this.

Reply:
Wrong change.
Should be `int(round(duration / 3600))`